frontend/public/components/cluster-settings/related-objects.tsx

```diff
@@ -0,0 +1,89 @@
+import * as React from 'react';
+import * as classNames from 'classnames';
+import * as _ from 'lodash-es';
+import { sortable } from '@patternfly/react-table';
+import { Table, TableRow, TableData, RowFunctionArgs } from '../factory';
+import {
+  referenceForModel,
+  ClusterOperator,
+  ClusterOperatorObjectReference,
+  useModelFinder,
+} from '../../module/k8s';
+import { ResourceLink } from '../utils';
+
+const tableColumnClasses = [
+  '',
+  classNames('pf-m-hidden', 'pf-m-visible-on-sm'),
+  classNames('pf-m-hidden', 'pf-m-visible-on-sm'),
```

Comment:
We should be able to fit at least two columns at mobile. Namespace might be more important to show since we have the badge for the kind.